scripts/tasks/sass-build.js

```diff
@@ -6,7 +6,7 @@ module.exports = function (gulp, plugins, consts) {
             .pipe(plugins.sass({outputStyle: 'compressed'})
                 .on('error', plugins.sass.logError))
             .pipe(plugins.autoprefixer({
-                browsers: ['last 2 versions', 'ie 8', 'ie 9'],
+                browsers: ['last 2 versions', 'OperaMini >= 5', 'Android >= 4', 'Chrome >= 28', 'Safari >= 7'],
```

Comment:
This translates to:

android 46
android 4.4.3-4.4.4
android 4.4
android 4.2-4.3
android 4.1
android 4
chrome 47
chrome 46
chrome 45
chrome 44
chrome 43
chrome 42
chrome 41
chrome 40
chrome 39
chrome 38
chrome 37
chrome 36
chrome 35
chrome 34
chrome 33
chrome 32
chrome 31
chrome 30
chrome 29
chrome 28
edge 13
edge 12
firefox 43
firefox 42
ie 11
ie_mob 11
ie_mob 10
ios_saf 9.0-9.2
ios_saf 8.1-8.4
op_mini 5.0-8.0
opera 34
opera 33
safari 9
safari 8
safari 7.1
safari 7